Extensions/Tracing-Zipkin.md

```diff
@@ -0,0 +1,54 @@
+
+[p]:  ../Protocol.md#frame-payload
+[rc]: ../Protocol.md#frame-request-channel
+[rf]: ../Protocol.md#frame-fnf
+[rr]: ../Protocol.md#frame-request-response
+[rs]: ../Protocol.md#frame-request-stream
+
+### Metadata Contents
+```
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |P|S|D|
+    +-+-+-+---------------------------------------------------------+
+    |                                                               |
+    +                                                               +
+    |                                                               |
+    +                           Trace ID                            +
+    |                                                               |
+    +                                                               +
+    |                                                               |
+    +---------------------------------------------------------------+
+    |                                                               |
+    +                           Span ID                             +
+    |                                                               |
+    +---------------------------------------------------------------+
+    |                                                               |
+    +                        Parent Span ID                         +
+    |                                                               |
+    +-+-+-----------------------------------------------------------+
+    |S|D|
+    +-+-+-----------------------------------------------------------+
+```
+
+* **Flags**: (3 bits)
+  * (**P**)arent Span Id: Tracing payload contains a parent span id.
+  * (**S**)ampling Decision: Tracing payload contains a sample value. (Exclusive with D flag.)
+  * (**D**)ebug: Tracing payload contains a debug value. (Exclusive with S flag.)
+* **Trace ID**: (128 bits) Unsigned 128-bit integer ID of the trace. Every span in a trace shares this ID.
+* **Span ID**: (64 bits) Unsigned 64-bit integer ID for a particular span. This may or may not be the same as the trace id.
+* **Parent Span ID**: (64 bits) Unsigned 64-bit integer ID for a particular parent span.  This is an optional ID that will only be present on child spans. That is the span without a parent id is considered the root of the trace. (Not present if P flag is not set)
+* **Sampling Decision**: The span should be reported to the tracing system. (Not present if S flag is not set)
```

Comment:
I wonder whether these flags should be combined with the first set into a single byte, and it made clear that the first byte is just the flags.

Reply:
@yschimke Because the sampling decision and debug flags are ternary (yes, no, and if unset defer to previous) they can’t be represented by a single bit.  Do you think it’s clear enough what’s going on if I end up with two bits for each of those in the flags section?  That’s quite inconsistent with everything else in the protocol.